packages/theme/src/theme/_modals.scss

```diff
@@ -5,12 +5,42 @@
 	flex-direction: column;
 }
 
+/** IE-FIX */
+.modal-dialog {
+	display: flex;
+	flex-direction: row;
+	flex-wrap: nowrap;
+
+	>.modal-content {
```

Comment:
This is supposed to style the same elements as the `.modal-content` below. Can you merge them ?